charts/stable/anonaddy/templates/_secrets.tpl

```diff
@@ -0,0 +1,16 @@
+{{/* Define the secrets */}}
+{{- define "anonaddy.secrets" -}}
+{{- $secretName := (printf "%s-anonaddy-secrets" (include "tc.v1.common.lib.chart.names.fullname" $)) }}
+
+{{- $appKey := randAlphaNum 64 -}}
+{{- $secretKey := randAlphaNum 64 -}}
+
+ {{- with lookup "v1" "Secret" .Release.Namespace $secretName -}}
+   {{- $authKey = index .data "APP_KEY" | b64dec -}}
+   {{- $secretKey = index .data "ANONADDY_SECRET" | b64dec -}}
+ {{- end }}
+enabled: true
+data:
+  APP_KEY: {{ $appKey }}
```

Comment:
```suggestion
  APP_KEY: {{ $appKey | b64enc }}
```
apparently it should be b64 encoded
https://github.com/anonaddy/docker/blob/master/README.md?plain=1#L107

Reply:
we dont add it there...its implied.

Reply:
are you sure ? `randAlphaNum` does not say the output is b64 
https://helm.sh/docs/chart_template_guide/function_list/#randalphanum-randalpha-randnumeric-and-randascii

Reply:
i know k8s secrets are but k8s might be decoding it an exposing it as clear and thats the problem ?

Reply:
This is not your every day secret. Its parsed from a common lib that does lots of things